src/gui/layertree/qgslayertreemapcanvasbridge.cpp

```diff
@@ -115,8 +115,12 @@ void QgsLayerTreeMapCanvasBridge::setCanvasLayers()
     switch ( projectCrsBehavior )
     {
       case QgsGui::UseCrsOfFirstLayerAdded:
-        QgsProject::instance()->setCrs( mFirstCRS, true );
+      {
+        const bool planimetric = QgsSettings().value( QStringLiteral( "/qgis/measure/planimetric" ), true ).toBool();
```

Comment:
Note to self - we should consider a "QgsProjectDefaultSettings" class or the like, to avoid scattered code like this...

Reply:
Wouldn't that be covered by the grant proposal for settings by @3nids? (Can't find it right now)

Reply:
It wasn't successful though, was it?

Reply:
No ...